package.json

```diff
@@ -51,4 +51,4 @@
     "semi": false,
     "singleQuote": true
   }
-}
+}
```

Comment:
could you please setup your json-editor to always add \n at the end?

Reply:
I think i was set "format on save by prettier, use prettier config in the repo", let me double check

Reply:
ah i didn't turn on for json files